src/Symfony/Component/Process/Process.php

```diff
@@ -338,10 +338,20 @@ public function start(callable $callback = null, array $env = []): void
             throw new RuntimeException(sprintf('The provided cwd "%s" does not exist.', $this->cwd));
         }
 
-        $process = @proc_open($commandline, $descriptors, $this->processPipes->pipes, $this->cwd, $envPairs, $this->options);
+        $lastError = 'unknown reason';
+        set_error_handler(function ($type, $msg) use (&$lastError) {
+            $lastError = $msg;
+
+            return true;
+        });
+        try {
+            $process = @proc_open($commandline, $descriptors, $this->processPipes->pipes, $this->cwd, $envPairs, $this->options);
+        } finally {
+            restore_error_handler();
+        }
 
         if (!\is_resource($process)) {
-            throw new RuntimeException('Unable to launch a new process.');
+            throw new RuntimeException('Unable to launch a new process: '.$lastError);
```

Comment:
Should we throw a `ProcessFailedException` here? That would improve backwards compatibility I think.

Or should we maybe create a new exception like `ProcessStartFailedException`?

Reply:
ProcessFailedException would be fine to me

Reply:
I created a new `ProcessStartFailedException` that extends the `ProcessFailedException` in 833bf1d2dbd88559d84d8528aa825c45df1bb86b.
This is a clean API I think and makes the changed behavior in PHP 8.3 a little bit more BC.